### ISSUE TITLE
Default Sorting

### ISSUE THREAD

**Issue by andyjhall** [rstaib/jquery-bootgrid#301]
How do i choose which row the gird defaults its sorting on? and once this is done how can i (in code) change the sorting based on a row name?

Thanks,

Andy.

**Comment by Iodine-** [comment]
Do you mean column sorting?

**Comment by RajeshDixitCF** [comment]
@Iodine,

I have a similar issue where my grid has 4k+ rows and on clicking on any column, it goes and fetches new data again. Is there a way to just sort rendered rows?

Also is there a way to have a custom sort function? I read sort method, but it expects an object and not a function.

**Comment by Iodine-** [comment]
@RajeshDixitCF Would you mind making a new issue for the above?  I would like to be able to track it separately of this issue as they are unrelated
